.idea/vcs.xml

```diff
@@ -2,5 +2,7 @@
 <project version="4">
   <component name="VcsDirectoryMappings">
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/ivy_tests/test_array_api" vcs="Git" />
```

Comment:
please remove `.idea` changes from the commit

Reply:
Didn't mean to add this file. Must have been added accidentally when I committed the changes. Nonetheless, I'll rollback the changes